### ISSUE TITLE
Sign up and reset account don't display correct error messages

### ISSUE THREAD

**Issue by pfortuna** [jsfiddle/jsfiddle-issues#1160]
When signing up, it returns what I assume is an error: "username". The error message is hard to see because Its a black font on a dark background. 
This error could mean that the username is already taken, but no matter what username I try, I get the same result. 

I got similar errors when using the reset account feature.

Sign in might be broken too but that I would need to have a valid user to verify it.

**Comment by oskarkrawczyk** [comment]
Errors are broken right now, not sure how to fix them for the moment. Will see what can be done (including styling).

That said the forms themselves work fine as far as I can see (there are hundreds of signups daily).

**Comment by andelus** [comment]
Hi, 
Neither the signup or the login are working, I did not receive anything after registration and when I tried to login It did not authenticate. Additionally, no error is shown beside _all_ when you log in, and email when you signup.
I checked the network requests everything seems fine on the server side, request wise other than that I don't know what is going on. 
<img width="498" alt="screen shot 2018-02-10 at 7 48 04 pm" src="https://user-images.githubusercontent.com/8386539/36065478-df5d2b72-0e9b-11e8-8a1f-9ec8905f3b96.png">
